channeldb/migration27/migration_test.go

```diff
@@ -0,0 +1,148 @@
+package migration27
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+
+	mig25 "github.com/lightningnetwork/lnd/channeldb/migration25"
+	mig26 "github.com/lightningnetwork/lnd/channeldb/migration26"
+	mig "github.com/lightningnetwork/lnd/channeldb/migration_01_to_11"
+	"github.com/lightningnetwork/lnd/channeldb/migtest"
+	"github.com/lightningnetwork/lnd/kvdb"
+)
+
+var (
+	// Create dummy values to be stored in db.
+	dummyPrivKey, _ = btcec.NewPrivateKey()
+	dummyPubKey     = dummyPrivKey.PubKey()
+	dummyOp         = wire.OutPoint{
+		Hash:  chainhash.Hash{},
+		Index: 9,
+	}
+
+	// testChannel is used to test the balance fields are correctly set.
+	testChannel = &OpenChannel{
+		OpenChannel: mig26.OpenChannel{
```

Comment:
Yo dawg, I herd you like `OpenChannels` 😆

Reply:
lol, image doing this for migration 100...